code/game/objects/effects/anomalies.dm

```diff
@@ -421,6 +423,83 @@
 			if(EXPLODE_LIGHT)
 				SSexplosions.lowturf += T
 
+/obj/effect/anomaly/delimber
+	name = "delimber anomaly"
+	icon_state = "delimber_anomaly"
+	aSignal = /obj/item/assembly/signaler/anomaly/delimber
+	immortal = TRUE
+	/// Time passed since the last effect, increased by delta_time of the SSobj
+	var/ticks = 0
```

Comment:
see my problem is that release delay is in seconds, ticks is in deciseconds, and ticks doesn't properly describe itself but is rather only describing the measurement of time